packages/g/glycin-loaders/package.yml

```diff
@@ -0,0 +1,30 @@
+name       : glycin-loaders
```

Comment:
I feel like `glycin` is a better name.

Reply:
I matched it with source name and fedora named it this too.  `glycin` is definitely shorter. :smile: